docs/sources/developers/plugins/data-frames.md

```diff
@@ -146,7 +146,7 @@ Dimensions: 4 fields by 4 rows
 +---------------------+-----------------+-----------------+----------------+
 ```
 
-can be converted into a data frame in wide format:
+Convert the above table into a data frame in wide format like this:
```

Comment:
How does the user convert the data format? Or is this this illustrative of the end result in which case the text should read closer to the original

Reply:
Good catch, I've commited a fix.